client/src/components/CardContainer/CardContainer.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {NavLink} from "react-router-dom";
+import {Swiper, SwiperSlide} from "swiper/react";
+import {Navigation} from 'swiper'
+import CardItem from '../CardItem/CardItem';
```

Comment:
Suggest adding CardItem component to` index.js` in `./components` and import this component from there